src/relayserver/ReplenishFunction.ts

```diff
@@ -0,0 +1,57 @@
+import { PrefixedHexString } from 'ethereumjs-tx/dist/types'
+import { toBN, toHex } from 'web3-utils'
+import { defaultEnvironment } from '../common/Environments'
+import { RelayServer } from './RelayServer'
+import { ServerAction } from './StoredTransaction'
+import { SendTransactionDetails } from './TransactionManager'
+
+export async function replenishStrategy (relayServer: RelayServer, workerIndex: number, currentBlock: number): Promise<PrefixedHexString[]> {
+  let transactionHashes: PrefixedHexString[] = []
+  if (relayServer.isCustomReplenish()) {
+    // If custom replenish is settled, here should be a call to a custom function for replenish workers strategy.
+    // Delete the next error if a custom replenish fuction is implemented.
+    throw new Error('No custom replenish function found, to remove this error please add the custom replenish implementation here deleting this line.')
+  } else {
+    transactionHashes = transactionHashes.concat(await defaultReplenishFunction(relayServer, workerIndex, currentBlock))
```

Comment:
No need to concat an empy array. Directly assigning the await defaultRep.... is the same